config/checker_guideline_map.json

```diff
@@ -0,0 +1,360 @@
+      "sei-cert": ["int31-c"]
+    },
+    "bugprone-undefined-memory-manipulation": {
+      "sei-cert": ["oop57-cpp"]
+    },
+    "bugprone-unhandled-self-assignment": {
+      "sei-cert": ["oop54-cpp"]
+    },
+    "bugprone-unused-return-value": {
+      "sei-cert": ["exp63-cpp"]
+    },
+    "cert-dcl03-c": {
+      "sei-cert": ["dcl03-c"]
+    },
+    "cert-dcl16-c": {
+      "sei-cert": ["dcl16-c"]
+    },
+    "cert-dcl37-c": {
+      "sei-cert": ["dcl37-c"]
+    },
+    "cert-dcl21-cpp": {
+      "sei-cert": ["dcl21-cpp"]
+    },
+    "cert-dcl50-cpp": {
+      "sei-cert": ["dcl50-cpp"]
+    },
+    "cert-dcl51-cpp": {
+      "sei-cert": ["dcl51-cpp"]
+    },
+    "cert-dcl54-cpp": {
+      "sei-cert": ["dcl54-cpp"]
+    },
+    "cert-dcl58-cpp": {
+      "sei-cert": ["dcl58-cpp"]
+    },
+    "cert-dcl59-cpp": {
+      "sei-cert": ["dcl59-cpp"]
+    },
+    "cert-env33-c": {
+      "sei-cert": ["env33-c"]
+    },
+    "cert-err09-cpp": {
+      "sei-cert": ["err09-cpp", "err61-cpp"]
+    },
+    "cert-err34-c": {
+      "sei-cert": ["err34-c"]
+    },
+    "cert-err52-cpp": {
+      "sei-cert": ["err52-cpp"]
+    },
+    "cert-err58-cpp": {
+      "sei-cert": ["err58-cpp"]
+    },
+    "cert-err60-cpp": {
+      "sei-cert": ["err60-cpp"]
+    },
+    "cert-err61-cpp": {
+      "sei-cert": ["err61-cpp"]
+    },
+    "cert-fio38-c": {
+      "sei-cert": ["fio38-c"]
+    },
+    "cert-flp30-c": {
+      "sei-cert": ["flp30-c"]
+    },
+    "cert-msc30-c": {
+      "sei-cert": ["msc30-c", "msc50-cpp"]
+    },
+    "cert-msc32-c": {
+      "sei-cert": ["msc32-c", "msc51-cpp"]
+    },
+    "cert-msc50-cpp": {
+      "sei-cert": ["msc50-cpp", "msc30-c"]
+    },
+    "cert-msc51-cpp": {
+      "sei-cert": ["msc51-cpp"]
+    },
+    "cert-mem57-cpp": {
+      "sei-cert": ["mem57-cpp"]
+    },
+    "cert-oop11-cpp": {
+      "sei-cert": ["oop11-cpp"]
+    },
+    "cert-oop54-cpp": {
+      "sei-cert": ["oop54-cpp"]
+    },
+    "cert-oop57-cpp": {
+      "sei-cert": ["oop57-cpp"]
+    },
+    "cert-oop58-cpp": {
+      "sei-cert": ["oop58-cpp"]
+    },
+    "cert-pos44-c": {
+      "sei-cert": ["pos44-c"]
+    },
+    "cert-str34-c": {
+      "sei-cert": ["str34-c"]
+    },
+    "core.CallAndMessage": {
+      "sei-cert": ["dcl41-c", "exp30-c", "exp33-c", "exp34-c", "exp39-c",
+                   "exp50-cpp", "exp53-cpp", "exp54-cpp", "exp57-cpp"]
+    },
+    "core.DivideZero": {
+      "sei-cert": ["int33-c"]
+    },
+    "core.NonNullParamChecker": {
+      "sei-cert": ["exp34-c", "exp34-c", "mem34-c"]
+    },
+    "core.NullDereference": {
+      "sei-cert": ["arr30-c", "dcl38-c"]
+    },
+    "core.StackAddressEscape": {
+      "sei-cert": ["dcl30-c", "exp54-cpp"]
+    },
+    "core.UndefinedBinaryOperatorResult": {
+      "sei-cert": ["exp33-c", "exp36-c", "exp53-cpp", "int32-c", "int34-c",
+                   "oop53-cpp"]
+    },
+    "core.uninitialized.Assign": {
+      "sei-cert": ["oop54-cpp"]
```

Comment:
Remove oop54-cpp as the finding is indirectly relevant to the point of this rule.